app/Files/File.php

```diff
@@ -0,0 +1,2 @@
+<?php
```

Comment:
Bob Martin did talked about this.  
Do you keep your room tidy all time or do you wait until it gets dirty (complicated) ?  
single responsibility principle.